src/main/java/parsing/SocketCommand.java

```diff
@@ -120,10 +91,10 @@ protected void setLogLevel() {
 	}
 	
 	private InetSocketAddress errorSimulatorAddress() throws UnknownHostException {
-		return new InetSocketAddress(InetAddress.getByName(tokens.get(SERVER_ADDR_IDX)), ERROR_PORT);
+		return new InetSocketAddress(InetAddress.getByName(tokens.get(SERVER_ADDR_IDX)), GLOBAL_CONFIG.SIMULATOR_PORT);
```

Comment:
Isn't the error simulator on the client and not on the server side?

" Your error simulator
may be single-threaded and is expected to work with one client only. To test multiple clients when using just
one machine, send requests directly to port 69. (Once we move to multiple machines, it's recommended that the
error simulator runs on the same computer as the client.)"

Reply:
Yeah Irusha! Good catch! Should also probably add a test for this as well. I just modified the code to use the configuration property for simulator port, so I didn't even see that!